dialplan/asterisk/extensions_lib_context.conf

```diff
@@ -7,3 +7,28 @@ same  = n,GotoIf($[${EXISTS(${WAZO_ENTRY_CONTEXT})}]?return:)
 same  = n,Set(__WAZO_ENTRY_CONTEXT=${WAZO_BASE_CONTEXT})
 same  = n,Set(__WAZO_ENTRY_EXTEN=${XIVO_BASE_EXTEN})
 same  = n(return),Return
+
+[callcounter]
+; ARG1 = caller user ID
+; ARG2 = destination user ID, if applicable (optional)
+exten = s,1,NoOp(Counting calls for ${ARG1}@WAZO_USER)
+same  =   n,NoOp(===================================)
+same  =   n,NoOp(currently: ${GROUP_COUNT(${ARG1}@WAZO_USER)} - maximum: ${WAZO_CALLER_SIMULTCALLS})
+same  =   n,NoOp(===================================)
+same  =   n,GotoIf($["${GROUP_COUNT(${ARG1}@WAZO_USER)}" >= "${WAZO_CALLER_SIMULTCALLS}"]?caller_full)
+same  =   n,Set(GROUP(WAZO_USER)=${ARG1})
+same  =   n,Set(OUTBOUND_GROUP_ONCE=${ARG1}@WAZO_USER)
+
+same  =   n,GotoIf($[${EXISTS(${ARG2})}]?:end)
+
+same  =   n,NoOp(Counting calls for destination ${ARG2}@WAZO_USER)
+same  =   n,NoOp(===================================)
+same  =   n,NoOp(currently: ${GROUP_COUNT(${ARG2}@WAZO_USER)} - maximum: ${WAZO_SIMULTCALLS})
```

Comment:
We can rename `WAZO_SIMULTCALLS` to `WAZO_CALLEE_SIMULTCALLS` to match `WAZO_CALLER_SIMULTCALLS`